pkg/deploy/lattice/listener_synthesizer.go

```diff
@@ -95,19 +102,18 @@ func (l *listenerSynthesizer) getSDKListeners(ctx context.Context) ([]*latticemo
 	var resService []*latticemodel.Service
 
 	err := l.stack.ListResources(&resService)
-	glog.V(6).Infof("service: %v ignore err: %v \n", resService, err)
+	l.log.Debugf("Ignoring error when listing services %s", err)
```

Comment:
if err then log error

Reply:
Will do in part 2 PR, thanks for catching this!